tasks.py

```diff
@@ -1,6 +1,9 @@
 from __future__ import print_function
 from invoke import task, run
 import os
+import yaml
+import shutil
+import pdb
```

Comment:
@blink1073 Not used? I use `shutil` and `yaml` in a few lines.  I keep `pdb` in there for debugging purposes, but I could remove it I suppose?